### PR TITLE
Add Espressif esp32s3 lcd ev v1.5

### PR DESCRIPTION
**Pull request by sanhu88**
Hi there,
Espressif release  v1.5 variety of ESP32-S3-LCD-EV-Board with hardware revision.
more details [here](https://docs.espressif.com/projects/esp-dev-kits/en/latest/esp32s3/esp32-s3-lcd-ev-board/user_guide.html#id1)

### CODE REVIEW

File: ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/sdkconfig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CONFIG_ESP_CONSOLE_UART_TX_GPIO=43
+CONFIG_ESP_CONSOLE_UART_RX_GPIO=44
+CONFIG_ESP_CONSOLE_UART_BAUDRATE=115200
+# end of ESP System Settings
+
+#
+# LWIP
+#
+CONFIG_LWIP_LOCAL_HOSTNAME="circuitpy"
+# end of LWIP
```
Comment (tannewt) [inline ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/sdkconfig:24]: You shouldn't need any of this.

File: ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,17 @@
+USB_VID = 0x303A
+USB_PID = 0x1001
```
Comment (tannewt) [inline ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/mpconfigboard.mk:2]: Please request a PID from here: https://github.com/espressif/usb-pids/blob/main/allocated-pids-espressif-devboards.txt

File: ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/sdkconfig (hunk 1 above)
Comment (tannewt) [reply ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/sdkconfig]: Sorry, I meant the file should be empty. The build fails with it missing.

Comment (sanhu88) [reply ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/sdkconfig]: Thank you, I am a newbie for build the Circuitpython.

File: ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/board.c [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+#include "shared-bindings/board/__init__.h"
+#include "shared-bindings/busio/I2C.h"
+#include "shared-bindings/dotclockframebuffer/DotClockFramebuffer.h"
+#include "shared-bindings/dotclockframebuffer/__init__.h"
+#include "shared-bindings/framebufferio/FramebufferDisplay.h"
+#include "shared-bindings/microcontroller/Pin.h"
+#include "shared-module/displayio/__init__.h"
+#include "boards/espressif_esp32s3_lcd_ev_v1.5/board.h"
+
+#define MP_DEFINE_BYTES_OBJ(obj_name, bin) mp_obj_str_t obj_name = {{&mp_type_bytes}, 0, sizeof(bin) - 1, (const byte *)bin}
```
Comment (tannewt) [inline ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/board.c:18]: This is breaking the CI. Have you built and tested this locally?

Comment (sanhu88) [reply ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/board.c]: Not yet. 
I update  macro name as commit 4a5624a by @dhalbert.

**Review by tannewt — APPROVED** [review]
Ok! Thanks for the update

**Pull request merged**